inmaputil/download_test.go

```diff
@@ -0,0 +1,43 @@
+package inmaputil
+
+import "testing"
+
+func TestDownloadLocal (t *testing.T) {
+	if k := Download("/dev/null"); k != "/dev/null" {
+		t.Error("Expected /dev/null, got ", k);
+	}
+}
+
+func TestDownloadLocal2 (t *testing.T) {
+	// dummy directory
+	if k := Download("/blah/test/"); k != "/blah/test/" {
+		t.Error("Expected /blah/test/, got ", k);
+	}
+
+	t.Logf("Passed directory tests. ");
+}
+
+func TestDownloadRemoteFail (t *testing.T) {
+	if k := Download("http://blah/test/"); k != "http://blah/test/" {
+		t.Error("Expected http://blah/test/, got ", k);
+	}
+	t.Logf("Passed failed fetching test. ")
+}
+
+func TestDownloadRemoteDecompress (t *testing.T) {
+	if test.Short() {
+		return
+	}
+	if k := Download("https://github.com/spatialmodel/inmap/blob/master/inmap/testdata/testEmis.shp.zip?raw=true"); k != "temp/testEmis.zip_extracted/testEmis.shp" {
```

Comment:
We'll want to make sure this works in the case where there are multiple files in the zipfile and when the files are either within or not within a subdirectory in the shapefile.